docs/examples/benchmark.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
```

Comment:
Should we have a `benchmark` folder?

Reply:
I don't have a plan. Just want some quick functions to reduce duplications. Here is for examples.